forte/data/ontology/ontology_code_generator.py

```diff
@@ -1078,6 +1080,38 @@ def parse_entry(
 
         return entry_item, property_names
 
+    def parse_ndarray(
+        self,
+        manager: ImportManager,
+        schema: Dict,
+        att_name: str,
+        desc: str,
+    ):
+        ndarray_dtype = None
+        if SchemaKeywords.ndarray_dtype in schema:
+            # TODO: validate dtype
+            ndarray_dtype = schema[SchemaKeywords.ndarray_dtype]
+
+        ndarray_shape = None
+        if SchemaKeywords.ndarray_shape in schema:
+            # TODO: validate shape
+            ndarray_shape = schema[SchemaKeywords.ndarray_shape]
+
+        # TODO: Throw warning if either dtype or shape is missing
```

Comment:
plan to deal with this todo? I found that `todo` will simply be forgotten in the future. We either fix them here or create issues to solve them in the future.

Reply:
done.